File: ws/gualotuna/u2/WS21Inheritances/Zoo/src/main/java/ec/edu/espe/zoo/model/Monkey.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Brayan Gualotuña, Dev Dynasty, DCCO-ESPE
 */
public class Monkey extends Primate{

    public Monkey(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, int numberOfLegs) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, numberOfLegs);
    }

   
   
    

    @Override
    public void feed(int food) {
        System.out.println("Feeding a monkey");
    }
    
}

File: hw/segarra/u2/HW18ZooInPythonAndJavaScript/ec.edu.zooinjavascript.model/Monkey.js
import Primate from "./Primate.js"

export default class Monkey extends Primate{
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate){
        super(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate)
    }

    feed(){
        console.log('Feeding the monkey with bananas')
    }
}